modules/nix/linux-builder.nix

```diff
@@ -133,8 +135,8 @@ in
 
     systems = mkOption {
       type = types.listOf types.str;
-      default = [ "${stdenv.hostPlatform.uname.processor}-linux" ];
-      defaultText = literalExpression ''[ "''${stdenv.hostPlatform.uname.processor}-linux" ]'';
+      default = [ builderWithOverrides.nixosConfig.nixpkgs.hostPlatform.system ];
+      defaultText = literalExpression ''[ config.nix.linux-builder.package.nixosConfig.nixpkgs.hostPlatform.system " ]'';
```

Comment:
```suggestion
      defaultText = literalExpression ''[ config.nix.linux-builder.package.nixosConfig.nixpkgs.hostPlatform.system ]'';
```

Reply:
Could we have descriptive text in `defaultText` instead here, if it’s not possible to provide a correct expression? In this case the user configuration has an effect on the resulting `hostPlatform` so the claimed default isn’t really accurate. Another alternative would be to export `builderWithOverrides` as `config.nix.linux-builder.finalPackage` or something (I vaguely recall there being a naming convention for such options in NixOS, but not what it is).

Reply:
I'd be happy to put a description instead.